src/Sushi.Mediakiwi/Controllers/ContentController.cs

```diff
@@ -642,7 +647,7 @@ private async Task<(ContentItem contentItem, bool isFilled)> getContentItemFromF
                             {
                                 isFilled = true;
                                 var dateTimeValue = new DateTime(unitTime);
-                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo("nl-NL");
+                                System.Globalization.CultureInfo dateCulture = new System.Globalization.CultureInfo(culture);
                                 content.Text = dateTimeValue.ToString("dd-MM-yyyy HH:mm:ss", dateCulture);

```

Comment:
Same as before.
should be something like:
```suggestion
content.Text = dateTimeValue.ToString($"{dateCulture.DateTimeFormat.ShortDatePattern} {dateCulture.DateTimeFormat.ShortTimePattern}");

```